huaweicloud/services/acceptance/css/resource_huaweicloud_css_cluster_test.go

```diff
@@ -100,117 +136,114 @@ func testAccCssCluster_basic(rName string, nodeNum int, keepDays int, tag string
+
   backup_strategy {
     keep_days  = %d
     start_time = "00:00 GMT+08:00"
     prefix     = "snapshot"
+    bucket      = huaweicloud_obs_bucket.cssObs.bucket
+    agency      = "css_obs_agency"
+    backup_path = "css_repository/acctest"
   }
 
   tags = {
     foo = "%s"
     key = "value"
   }
 }
-
-`, testAccVpc(rName), rName, nodeNum, keepDays, tag)
+`, testAccCssBase(rName), rName, nodeNum, keepDays, tag)
 }
 
 func testAccCssCluster_security(rName string, nodeNum int, tag string) string {
 	return fmt.Sprintf(`
 %s
 
 resource "huaweicloud_css_cluster" "test" {
-  name            = "%s"
-  engine_version  = "7.9.3"
-  expect_node_num = %d
-  security_mode   = true
-  password        = "Test@passw0rd"
-
-  node_config {
-    flavor            = "ess.spec-4u16g"
-    availability_zone = data.huaweicloud_availability_zones.test.names[0]
-
-    network_info {
-      security_group_id = huaweicloud_networking_secgroup.test.id
-      subnet_id         = huaweicloud_vpc_subnet.test.id
-      vpc_id            = huaweicloud_vpc.test.id
+  name           = "%s"
+  engine_version = "7.10.2"
+  security_mode  = true
+  password       = "Test@passw0rd"
+
+  availability_zone = data.huaweicloud_availability_zones.test.names[0]
+  security_group_id = huaweicloud_networking_secgroup.test.id
+  subnet_id         = huaweicloud_vpc_subnet.test.id
+  vpc_id            = huaweicloud_vpc.test.id
+
+
+  period_unit = "month"
+  period      = %d
+  auto_renew  = "true"
+
+  ess_node_config {
+    flavor          = "ess.spec-4u8g"
+    instance_number = 1
+    volume {
+      volume_type = "HIGH"
+      size        = 40
     }
+  }
 
+  master_node_config {
+    flavor          = "ess.spec-4u8g"
+    instance_number = 3
     volume {
       volume_type = "HIGH"
       size        = 40
-    }  
+    }
   }
 
-  tags = {
-    foo = "%s"
-    key = "value"
+  client_node_config {
+    flavor          = "ess.spec-4u8g"
+    instance_number = 1
+    volume {
+      volume_type = "HIGH"
+      size        = 40
+    }
   }
-}
 
-`, testAccVpc(rName), rName, nodeNum, tag)
-}
+  cold_node_config {
+    flavor          = "ess.spec-4u8g"
+    instance_number = 1
+    volume {
+      volume_type = "HIGH"
+      size        = 40
+    }
+  }
 
-func testAccCheckCssClusterDestroy(s *terraform.State) error {
-	config := acceptance.TestAccProvider.Meta().(*config.Config)
-	client, err := config.CssV1Client(acceptance.HW_REGION_NAME)
-	if err != nil {
-		return fmtp.Errorf("error creating CSS client: %s", err)
-	}
+  vpcep_endpoint {
+    endpoint_with_dns_name = true
+  }
 
-	for _, rs := range s.RootModule().Resources {
-		if rs.Type != "huaweicloud_css_cluster" {
-			continue
-		}
+  public_access {
+    bandwidth         = 1
+    whitelist_enabled = false
 
-		_, err := cluster.Get(client, rs.Primary.ID)
-		if err == nil {
-			return fmtp.Errorf("css cluster still exists, cluster_id:%s", rs.Primary.ID)
-		}
-	}
+  }
 
-	return nil
-}
+  kibana_public_access {
```

Comment:
don't enalbe kibana public access in test, we should avoid open service port with eip address.